src/Serilog/project.json

```diff
@@ -31,7 +19,7 @@
         "dotnet5.1": {
             "compilationOptions": {
                 "keyFile": "../../assets/Serilog.snk",
-                "define": [ "NO_APPDOMAIN" ]
+                "define": [ "ASYNCLOCAL", "NO_APPDOMAIN" ]
```

Comment:
Is this missing a `"LOGCONTEXT"`? If so, is LogContext now supported on all target plaforms? It'd be great to get rid of the conditional if we can.